### DISCUSSION TITLE
Validation for RelationshipAssociations

### DISCUSSION THREAD

**Discussion by jifox** [nautobot/nautobot#492] [Q&A]
I want to assign the Gateway address to a prefix

When defining the relationship `ipam | prefix` to `ipam | IP address` I want to validate that the assigned IP address is within the prefix network.

How can this validation be achived?

> @dgarros 
>Unfortunately I don’t think this is possible today.
>If we could create a custom validator for RelationshipAssociation it would be possible but this is not available today.
>I would suggest to open a discussion in Github about that.

**Comment by glennmatthews** [comment]
Related: #498 (I know you also opened that, just adding the link here for the benefit of others!)
